### ISSUE TITLE
Errors being generated in debug log for alert 2052 (New alert)

### ISSUE THREAD

**Issue by robert81** [WPWhiteSecurity/WP-Security-Audit-Log#76]
P.S. I started using WordPress 4.2 for testing, it will be released next week hence all testing should be done on it (https://wordpress.org/news/2015/04/wordpress-4-2-beta-4/).

Each time I change the parent of a category, the following is being reported in the debug.log:

     [15-Apr-2015 16:08:22 UTC] PHP Notice:  Trying to get property of non-object in C:\wamp\htdocswp\wp-content\plugins\wp-security-audit-log\classes\Sensors\Content.php on line 429

     [15-Apr-2015 16:08:22 UTC] PHP Stack trace:

     [15-Apr-2015 16:08:22 UTC] PHP   1. {main}() C:\wamp\htdocswp\wp-admin\edit-tags.php:0

     [15-Apr-2015 16:08:22 UTC] PHP   2. wp_update_term() C:\wamp\htdocswp\wp-admin\edit-tags.php:145

     [15-Apr-2015 16:08:22 UTC] PHP   3. do_action() C:\wamp\htdocswp\wp-includes\taxonomy.php:3543

     [15-Apr-2015 16:08:22 UTC] PHP   4. call_user_func_array() C:\wamp\htdocswp\wp-includes\plugin.php:496

     [15-Apr-2015 16:08:22 UTC] PHP   5. WSAL_Sensors_Content->EventChangedCategoryParent() C:\wamp\htdocswp\wp-includes\plugin.php:496

**Comment by carmelopreeostudios** [comment]
"category_parent" is not used anymore in this version and this makes sense because
is exactly the same of "parent"

**Comment by robert81** [comment]
@carmelopreeostudios  I tested the new branch (https://github.com/WPWhiteSecurity/WP-Security-Audit-Log/tree/hotfix/category-parent-wordpressbeta) and although the above issue is fixed there is still another issue as shown below. Can we please make sure these basic tests are done before committing?

     [16-Apr-2015 09:28:23 UTC] PHP Notice:  Undefined property: WP_Error::$name in C:\wamp\htdocswp\wp-content\plugins\wp-security-audit-log\classes\Sensors\Content.php on line 426

     [16-Apr-2015 09:28:23 UTC] PHP Stack trace:

     [16-Apr-2015 09:28:23 UTC] PHP   1. {main}() C:\wamp\htdocswp\wp-admin\edit-tags.php:0

     [16-Apr-2015 09:28:23 UTC] PHP   2. wp_update_term() C:\wamp\htdocswp\wp-admin\edit-tags.php:145

     [16-Apr-2015 09:28:23 UTC] PHP   3. do_action() C:\wamp\htdocswp\wp-includes\taxonomy.php:3543

     [16-Apr-2015 09:28:23 UTC] PHP   4. call_user_func_array() C:\wamp\htdocswp\wp-includes\plugin.php:496

     [16-Apr-2015 09:28:23 UTC] PHP   5. WSAL_Sensors_Content->EventChangedCategoryParent() C:\wamp\htdocswp\wp-includes\plugin.php:496

**Comment by carmelopreeostudios** [comment]
@robert81 yes was my mistake, now I tested more

**Comment by robert81** [comment]
@carmelopreeostudios no problem. Looking good now. Let's merge to develop and then to master so we do the last phase of testing. Please advise once the merging is done.

**Comment by carmelopreeostudios** [comment]
Hi Robert,
the merging is done into the develop branch

Regards,
Carmelo

On 16 April 2015 at 12:48, Robert Abela <notifications@github.com> wrote:

> Closed #76
> <https://github.com/WPWhiteSecurity/WP-Security-Audit-Log/issues/76>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WPWhiteSecurity/WP-Security-Audit-Log/issues/76#event-282946982>
> .
>



-- 

*Carmelo Cutrera*
Web developer

*Mobile: *00356 99664357

*Office: *Entrance C, Level 2, Capital Business Centre, Zwejt Street, San
Gwann, SGN 3000. Malta
*Web:* www.preeostudios.com

<http://www.preeostudios.com/>
